schema.sql

```diff
@@ -32,4 +32,29 @@ ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE animals ADD COLUMN species_id INTEGER;
 ALTER TABLE animals ADD COLUMN owner_id INTEGER;
 ALTER TABLE animals ADD FOREIGN KEY (species_id) REFERENCES species(id);
-ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+/* Joint tables - visits */
+
+CREATE TABLE vets (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE specializations (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  species_id INT REFERENCES species(id),
+  vet_id INT REFERENCES vets(id),
+  PRIMARY KEY(id)
+);
```

Comment:
- [ ] In this table, we only need `species_id and vet_id` columns, so we don't need `ID` column